TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -81,11 +81,14 @@ class AllStopsTableViewController: UIViewController {
             for busStop in allStops {
```

Comment:
I know u didn't write this chunk of code but could we clean it up a little bit. `currentChar` seems to be unnecessary as we can use firstChar to filter into the correct letter. Let's remove the currentChar logic and just use if lets to check if the dictionary has an associated list with `firstChar`